clang-tools-extra/clang-tidy/utils/UseRangesCheck.cpp

```diff
@@ -164,6 +165,24 @@ void UseRangesCheck::registerMatchers(MatchFinder *Finder) {
 static void removeFunctionArgs(DiagnosticBuilder &Diag, const CallExpr &Call,
                                ArrayRef<unsigned> Indexes,
                                const ASTContext &Ctx) {
+  const auto GetCommaLoc = [&](SourceLocation BeginLoc,
```

Comment:
Nit suggestion: as far as I can see, we only need `Ctx` here, so no point in capturing all variables.
```suggestion
  const auto GetCommaLoc = [&Ctx](SourceLocation BeginLoc,
```